gradle/dependencies.gradle

```diff
@@ -17,26 +17,31 @@
 def versions = [
         androidTest: '0.5',
         androidxVersion: '1.0.0',
-        uberJava: '0.8.0',
+        uberJava: '0.8.1',
+]
+
+def network = [
+        moshi: 'com.squareup.moshi:moshi:1.9.2',
 ]
 
 def build = [
         gradleVersion: '4.9',
         buildToolsVersion: '28.0.2',
         compileSdkVersion: 28,
         ci: 'true' == System.getenv('CI'),
-        minSdkVersion: 15,
+        minSdkVersion: 16,
         targetSdkVersion: 28,
 
         repositories: [
                 plugins: 'https://plugins.gradle.org/m2/'
         ],
 
         gradlePlugins: [
-                android: 'com.android.tools.build:gradle:3.2.0',
+                android: 'com.android.tools.build:gradle:3.3.2',
                 release: 'net.researchgate:gradle-release:2.1.2',
                 github: 'co.riiid:gradle-github-plugin:0.4.2',
                 cobertura: 'net.saliman:gradle-cobertura-plugin:2.3.1',
+                buildConfig: 'gradle.plugin.de.fuerstenau:BuildConfigPlugin:1.1.8'
```

Comment:
What are we doing with this plugin? I don't see other uses, did I miss something?

Reply:
removed